physics/Interstitials/UFS_SCM_NEPTUNE/sfcsub.F

```diff
@@ -346,6 +352,10 @@ subroutine sfccycle(lugb,len,lsoil,sig1t,deltsfc                    &
       parameter(veglmx=1.0,veglmn=0.0,vegomx=0.0,vegomn=0.0,
      &          vegsmx=1.0,vegsmn=0.0,vegimx=0.0,vegimn=0.0,
      &          vegjmx=0.0,vegjmn=0.0)
+! leaf area index      
+      parameter(lailmx=7.0,lailmn=0.0,laiomx=7.0,laiomn=0.0,   ! Sanath fix these values
```

Comment:
in general, I wouldn't add your name in the code unless there is some precedent, e.g., in the header section

Reply:
I left that so that I can revisit and fix teh max and min values to any suggested value. As of now its an arbitrary 7 and 0